Use correct tolerations value when deploying cilium-operator and cilium-etcd-operator via helm

michaelpetrov opened the pull request:
<!-- Description of change -->

Use correct tolerations value when deploying cilium-operator via helm. Without this the global `tolerations` value is used and the operator runs on all nodes including ones with taints.


```release-note
Use correct tolerations value when deploying cilium-operator via helm.
```
Signed-off-by: Michael Petrov <petrov.michael@gmail.com>

michaelpetrov committed:
Use correct tolerations value when deploying cilium-operator via helm

<!-- Description of change -->

Use correct tolerations value when deploying cilium-operator via helm. Without this the global `tolerations` value is used and the operator runs on all nodes including ones with taints.

```release-note
Use correct tolerations value when deploying cilium-operator via helm.
```
Signed-off-by: Michael Petrov <petrov.michael@gmail.com>

kaworu reviewed (approved):
Hi @michaelpetrov and thanks for the PR. Patch LGTM.

> Without this the global tolerations value is used and the operator runs on all nodes including ones with taints.

Can you elaborate on this because I see no difference between the global tolerations and the operator's ones by default:
```console
$ yq '.tolerations,.operator.tolerations' install/kubernetes/cilium/values.yaml
[
  {
    "operator": "Exists"
  }
]
[
  {
    "operator": "Exists"
  }
]
```

michaelpetrov committed:
Use correct tolerations value when deploying cilium-etcd-operator via helm

<!-- Description of change -->

Use correct tolerations value when deploying cilium-etcd-operator via helm. Without this, the global `tolerations` value is used and the operator runs on all nodes including ones with taints. Currently the `values.yaml` file claims it is available (but has been unused so far)

```release-note
Use correct tolerations value when deploying cilium-etcd-operator via helm.
```

Signed-off-by: Michael Petrov <petrov.michael@gmail.com>

nathanjsweet reviewed (approved):
Good catch. Thanks for the PR!
